app/templates/road-sign-concepts/road-sign-concept.hbs

```diff
@@ -12,26 +12,36 @@
       <c.header @badgeSkin="brand">
         <div class="flex-content-between">
           <div class="no-flex-shrink">
-            <img class="au-u-margin-right" alt={{t 'abb.name'}} src={{@model.image}}>
+            <img class="au-u-margin-right" alt={{t 'abb.name' }} src={{@model.image}}>
```

Comment:
I don't think this change is needed?

Reply:
I think this is a result of auto formatting. Indeed not needed. Can be dropped if you like.